core/workflow-operator/src/main/scala/edu/uci/ics/amber/operator/sink/managed/ProgressiveSinkOpDesc.java

```diff
@@ -45,7 +48,7 @@ public class ProgressiveSinkOpDesc extends SinkOpDesc {
     private Option<String> chartType = Option.empty();
 
     @JsonIgnore
-    private SinkStorageReader storage = null;
+    private VirtualDocument<Tuple> storage = null;
```

Comment:
let's remove this from Desc in a future PR.

Reply:
Sure, added a TODO here.